File: ios/RCTMGL/RCTMGLCalloutManager.m
//
//  RCTMGLCalloutViewManager.m
//  RCTMGL
//
//  Created by Nick Italiano on 10/13/17.
//  Copyright © 2017 Mapbox Inc. All rights reserved.
//

#import "RCTMGLCalloutManager.h"
#import "RCTMGLCallout.h"

@implementation RCTMGLCalloutManager

RCT_EXPORT_MODULE()

- (UIView *)view
{
    return [[RCTMGLCallout alloc] init];
}

@end

File: ios/RCTMGL-v10/RCTMGLCalloutManager.swift
import Foundation
import MapboxMaps

@objc(RCTMGLCalloutManager)
class RCTMGLCalloutManager  : RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
    
    override func view() -> UIView! {
        return RCTMGLCallout()
    }
}
